packages/styles/scss/fonts/README.md

```diff
@@ -0,0 +1,134 @@
+# `scss/fonts`
+
+The `scss/fonts` folder provides a way to incorporate IBM Plex into your
+project. Below is a table of what fonts are currently supported from IBM Plex in
+Carbon:
+
+| Font                      | Available | Entrypoint                    |
+| ------------------------- | --------- | ----------------------------- |
+| IBM Plex Mono             | ✅        | `scss/fonts/mono`             |
+| IBM Plex Sans Arabic      | ✅        | `scss/fonts/sans-arabic`      |
+| IBM Plex Sans Devanagari  | ✅        | `scss/fonts/sans-devanagari`  |
+| IBM Plex Sans Hebrew      | ✅        | `scss/fonts/sans-hebrew`      |
+| IBM Plex Sans KR          |           |                               |
+| IBM Plex Sans JP          |           |                               |
+| IBM Plex Sans Thai Looped | ✅        | `scss/fonts/sans-thai-looped` |
+| IBM Plex Sans Thai        | ✅        | `scss/fonts/sans-thai`        |
+| IBM Plex Sans             | ✅        | `scss/fonts/sans`             |
+| IBM Plex Serif            | ✅        | `scss/fonts/serif`            |
+
+By default, Carbon provides the default font weights for: IBM Plex Mono, IBM
+Plex Sans, and IBM Plex Serif. To bring in additional fonts, you can include the
+`@carbon/styles/scss/fonts` entrypoint and configure it:
+
+```scss
+@use '@carbon/styles/scss/fonts' with (
+  $fonts: (
+    IBM-Plex-Sans-Arabic: true,
+  ),
+);
+```
+
+You can also configure it to disable specific fonts:
```

Comment:
If you disable a font here do you need to enable another one by default or will the other 2 default ones just take precedent ?

Reply:
@aledavila great question, basically there are the three fonts that are enabled by default. If you disable one of them, the others will still be enabled.

The trick for this is in the `fonts/_index.scss` file that merges our "default" fonts with the given `$fonts` map:

```scss
$-default-fonts: (
  IBM-Plex-Sans: true,
  IBM-Plex-Serif: true,
);

// This is the variable that callers would change with `@use`
$fonts: () !default;

// Merge the two maps, prefer values in what the user gives us over our default values
$fonts: map.merge($-default-fonts, $fonts);
```

Reply:
Oh cool. Thank !